corehq/apps/auditcare/tests/test_auth.py

```diff
@@ -28,7 +28,7 @@ def create_web_user(self):
         # A Couch user is required for logout
         user = WebUser.create('', USERNAME, 'mockmock', None, None)
         user.save()
-        self.addCleanup(lambda: user.delete(None))
+        self.addCleanup(user.delete, '', deleted_by=None)
```

Comment:
why not just pass in `None` instead of a blank string?

Reply:
We could. Just that the domain in this test is a blank string and i was simply passing domain in all tests. Not sure why we even allow that.